src/cameras/src/pm/Acquisition.cpp

```diff
@@ -300,7 +311,7 @@ pm::Acquisition<F, C>::Acquisition(std::shared_ptr<pm::Camera<F>> camera) : m_ca
     uint64_t framesMax = (0xFFFFFFFF - 1) / frameBytes;
 #endif
     //TODO parameterize min value
-    framesMax = std::min<uint64_t>(1000, framesMax); //Use 1000 frames max for now so startup isn't so slow
+    //framesMax = std::min<uint64_t>(1000, framesMax); //Use 1000 frames max for now so startup isn't so slow
     //const uint64_t recommendedFrameCount = std::min<uint64_t>(10 + std::min<uint64_t>(frameCount, framesMax), bufferCount);
```

Comment:
I can't test this today but if this helps or fixes the issue I'll clean this mess up